src/samVisualize/rendering/configConstants/NodeConfig.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * Configuration constants regarding Node rendering
+ */
+export class NodeConfig {
```

Comment:
Nit: probably doesn't need to be a class, just export consts.

Reply:
Yeah that works, and then `import * as NodeConfig from '...'`